polyfills/ElementTraversal/config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
The name here is a problem because `ElementTraversal` is not a web platform specced global. You can either use a descriptive name prefixed with `~` as we've done for `~html5-elements`, or use one of the real properties exported by the polyfill, eg `Element/protoype/firstElementChild` and list the others as `aliases` in the config.  This configuration option is documented in the contributors guide.

My preference would be for you to do the latter because then I can request any of the included features individually using their real API names in the `features` URL argument, eg `/v2/polyfill.js?features=Element.prototype.firstElementChild`.